### DISCUSSION TITLE
histplot: cumulative in reverse order

### DISCUSSION THREAD

**Discussion by FWin22** [mwaskom/seaborn#3224] [Q&A]
Hi,

would it be possible to allow cumulative to take a value of "-1" to enable the cumulative distribution to be calculated in reverse order. similar to the matplotlib.hist function (https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.hist.html).

**Comment by mwaskom** [comment]
That's a pretty hacky API, imo. I'm not firmly against a complementary cumulative distribution in `histplot` but I would be interested to know when you'd want to use a histogram for that rather than a ECDF.

**Reply by FWin22** [reply]
Hi @mwaskom,
thanks for the reply. I didn't check the ECDF function. In fact it does somehow what i want, but the visuals are not as nice, as in the histplot. Especially if you want a bar-chart with a small number of discrete values.

To give you some examples, where i believe the histogram is the better choice:

- scores in sports (e.g. in basketball: display the number of 50+/40+/30+/... games a player has)
- meteorology (e.g. displaying the number of days that exceed a certain temperature threshold or precipitation

In these scenarios a "complementary" cummulative histogram would be very helpful.

**Comment by LukasMueller187** [comment]
Yep, please add this feature as provided in matplotlib. Example would be an abbot curve (horizontal hist plots), e.g.:
![image](https://github.com/mwaskom/seaborn/assets/30463947/ce4e1e71-335e-431b-8c4b-a574af8c3277)

Seaborn output unfortunately:
![image](https://github.com/mwaskom/seaborn/assets/30463947/925865cf-d761-4dea-9497-9c09b2fe8c85)
